res/css/style.css

```diff
@@ -39,6 +39,19 @@ body {
   color: var(--grey-90);
 }
 
+@media (forced-colors: active) {
+  :root {
+    /* The following properties are retrieved in index.js to create SVG filters that
+    * we can apply on icons to control their color.
+    * These variable can be declared in any rule, as long as their value is still
+    * a valid color (which means we can use `light-dark()` or any other color functions).
+    * The colors can be applied on SVG icons with `filter: url(#--var-name)` */
+    --button-icon-color: ButtonText;
+    --button-icon-hover-color: SelectedItem;
+    --button-icon-active-color: SelectedItem;
```

Comment:
BTW the same than usual still happens: SelectedItem is not great for me in light mode. But out of scope here...

Reply:
yeah, I need to reach to the accessibility team to sort this out